.travis_suppressions

```diff
@@ -2,12 +2,10 @@ unusedPrivateFunction:test/testbufferoverrun.cpp
 unusedPrivateFunction:test/testcmdlineparser.cpp
 redundantNextPrevious:test/testtoken.cpp
 simplePatternError:test/testtoken.cpp
-noValidConfiguration
 shadowFunction
 functionConst
 functionStatic
 bitwiseOnBoolean
-hidingInheritedPublic
```

Comment:
Did not happen when I ran the checks locally.

Reply:
I am surprised.. I thought we would get some unused suppression warning. maybe that is not enabled in self checks. somebody should investigate and try to enable that.